config/webpack.config.js

```diff
@@ -96,6 +102,11 @@ var configGenerator = (options) => {
               NODE_ENV: JSON.stringify(process.env.NODE_ENV)
           }
       }),
+      new sassLintPlugin({
+        configFile: 'config/.sass-lint.yml',
```

Comment:
Erp, that was a mistake.  It should just be pointing at `config/sass-lint.yml` (which already exists in master).